api/v1alpha/api/openapi.yaml

```diff
@@ -49004,7 +49035,49 @@ components:
           readOnly: true
           type: string
         eventTypeName:
-          $ref: '#/components/schemas/DefaultAlertViewForNdsGroup_eventTypeName'
+          description: Incident that triggered this alert.
```

Comment:
[Unrelated to pr]
Coming from the point of doing generally nonharmful transformations would we consider adding prefix to each enum. This way any overwrite will be persistent. 

It doesn't affect SDK in any way as it does gets ignored but it will affect docs and other tools reading that schema.

WDYT about this? Is that jira worthy?